src/main/webapp/app/entities/flair-bi/flair-bi.html

```diff
@@ -180,10 +188,12 @@
                     </div>
                     <div ng-style="{'opacity': v.bodyProperties.opacity}"
                         ng-if="$root.vizualizationServiceMode == 'grpc'" id="content-{{v.visualBuildId}}"
-                        class="printme content widget-content" visualization-render-grpc can-build="v.canBuild()" data="v" id="v.id || v.visualBuildId" widget="{{v.metadataVisual.functionname}}"
+                        class="printme content widget-content" visualization-render-grpc can-build="v.canBuild()"
+                        data="v" id="v.id || v.visualBuildId" widget="{{v.metadataVisual.functionname}}"
```

Comment:
There is no change in this line.. How would you identify which line got change?..pls ensure only changed line should be visible here otherwise it is difficult to trace the change